src/components/Contacts/ContactsMap/ContactsMap.tsx

```diff
@@ -10,6 +10,7 @@ import {
 } from '@react-google-maps/api';
 import { useTranslation } from 'react-i18next';
 import { StatusEnum } from 'src/graphql/types.generated';
+import { useContactPartnershipStatuses } from 'src/hooks/useContactPartnershipStatuses';
```

Comment:
When I click on a pin, I'm seeing two popups, and one is empty.

<img width="679" alt="Screenshot 2024-08-15 at 9 03 50 AM" src="https://github.com/user-attachments/assets/d61b5c9b-4a71-4345-bda8-5ff1b38a70de">

Reply:
This is taking a bit to figure out but I'm working on it.

Reply:
Fixed.